pkg/conf/model.go

```diff
@@ -128,3 +130,8 @@ type Infra struct {
 	AWSSecretAccessKey string `yaml:"awsSecretAccessKey,omitempty"`
 	AWSRegion          string `yaml:"awsRegion,omitempty"`
 }
+
+
+// name of the plugin that should match with it's docker profile
+// this will be used with the REGISTER_PLUGINS env var
+type PluginName = string
```

Comment:
What's the difference between `type PluginName = string` and `type PluginName string`?

Reply:
I asked chatgpt and got an answer, I guess the real question is whether it matters if we're using one form or the other given that other type declarations in the file omit the `=`

Reply:
Ah so i made a type alias so PluginName is just a string but when reading the code it's a bit more explicit. The latter is a new type so if we wanted to add methods to PluginName alongside methods that string already has we could do it that way

Reply:
in this case there's not a huge diff